inc/functions-icons.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * SVG icons related functions and filters.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Seventeen
+ * @since 1.0
+ */
+ 
+/**
+ * Add SVG definitions to the footer.
+ */
+function twentyseventeen_include_svg_icons() {
+	
+	// Define SVG sprite file.
+	$svg_icons = get_template_directory() . '/assets/images/svg-icons.svg';
+	
+	// If it exists, include it.
+	if ( file_exists( $svg_icons ) ) {
+		echo file_get_contents( $svg_icons );
```

Comment:
Not sure how I feel about `file_get_contents()` within this theme. While not as semantic, seems like it would be more efficient to put the svg data within PHP instead.